ddp-workspace/projects/ddp-sdk/src/lib/services/serviceAgents/participantProfileServiceAgent.service.ts

```diff
@@ -0,0 +1,37 @@
+import { Inject, Injectable } from '@angular/core';
+import { HttpClient } from '@angular/common/http';
+import { Observable } from 'rxjs';
+import { ConfigurationService, LoggingService, SessionMementoService, UserProfile } from 'ddp-sdk';
+import { SessionServiceAgent } from './sessionServiceAgent.service';
+
+@Injectable()
+export class ParticipantProfileServiceAgent extends SessionServiceAgent<any> {
+    constructor(
+        session: SessionMementoService,
+        @Inject('ddp.config') configuration: ConfigurationService,
+        http: HttpClient,
+        logger: LoggingService) {
+        super(session, configuration, http, logger, null);
+    }
+
+    public updateParticipantProfiles(participants: Array<{ guid: string; profile: any}>): Observable<any>[] {
+        return participants.map(participant => {
+            const {guid, profile} = participant;
+            return this.updateParticipantProfile(guid, profile);
+        });
+    }
+
+    public updateParticipantProfile(guid, profile: UserProfile): Observable<any> {
```

Comment:
`updateParticipantProfile` is only used in this service. 
Probably petter to make it `private`.

Reply:
Fixed